evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -152,7 +152,9 @@ def _format_dates(self, dates, X, y, predict=False):
             forecasting_ = import_or_raise(
                 "sktime.forecasting.base", error_msg=arima_model_msg
             )
-            fh_ = forecasting_.ForecastingHorizon(dates, is_relative=False)
+            fh_ = forecasting_.ForecastingHorizon(
+                [i + 1 for i in range(len(dates))], is_relative=True
+            )
```

Comment:
So, dumb question, it seems that right now relative forecasting is all we support with this change.  Is there any benefit to changing the api by having `is_relative` be assigned during the regressor's creation?   I'm not super familiar, so maybe we only want relative forecasting, but it seems to the luddite like me that "both" might be desirable.

Reply:
Not dumb at all and this is certainly something we can discuss further. Based on the time series conversations that have been going on, we want to make sure that however we implement this, it should work well with the moving window strategy we want to leverage in Rolling Origin Cross Validation. I think especially [considering](https://alteryx.atlassian.net/wiki/spaces/PS/pages/931004419/Draft+Version+1+of+Time+Series+AutoMLSearch+API#In-depth-look-of-pipeline.predict-for-the-test-set) `gap` and `forecast_horizon`, we'd want to look a certain number of steps ahead when making our predicts. Having relative forecasting would make this easier since we can determine how far ahead we want to look, and what those indices should be.